Extension/package.json

```diff
@@ -1,6094 +1,6095 @@
-              "cwd": {
-                "type": "string",
-                "description": "%c_cpp.debuggers.cwd.description%",
-                "default": "${workspaceRoot}"
-              },
-              "environment": {
-                "type": "array",
-                "description": "%c_cpp.debuggers.environment.description%",
-                "items": {
-                  "type": "object",
-                  "default": {},
-                  "properties": {
-                    "name": {
-                      "type": "string"
-                    },
-                    "value": {
-                      "type": "string"
-                    }
-                  }
-                },
-                "default": []
-              },
-              "envFile": {
-                "type": "string",
-                "description": "%c_cpp.debuggers.envFile.description%",
-                "default": "${workspaceFolder}/.env"
-              },
-              "symbolSearchPath": {
-                "type": "string",
-                "description": "%c_cpp.debuggers.symbolSearchPath.description%",
-                "default": ""
-              },
-              "stopAtEntry": {
-                "type": "boolean",
-                "description": "%c_cpp.debuggers.stopAtEntry.description%",
-                "default": false
-              },
-              "dumpPath": {
-                "type": "string",
-                "description": "%c_cpp.debuggers.dumpPath.description%",
-                "default": ""
-              },
-              "visualizerFile": {
-                "type": "string",
-                "description": "%c_cpp.debuggers.cppvsdbg.visualizerFile.description%",
-                "default": ""
-              },
-              "externalConsole": {
-                "type": "boolean",
-                "description": "%c_cpp.debuggers.cppvsdbg.externalConsole.description%",
-                "default": false
-              },
-              "console": {
-                "type": "string",
-                "enum": [
-                  "internalConsole",
-                  "integratedTerminal",
-                  "externalTerminal",
-                  "newExternalWindow"
-                ],
-                "enumDescriptions": [
-                  "%c_cpp.debuggers.cppvsdbg.console.internalConsole.description%",
-                  "%c_cpp.debuggers.cppvsdbg.console.integratedTerminal.description%",
-                  "%c_cpp.debuggers.cppvsdbg.console.externalTerminal.description%",
-                  "%c_cpp.debuggers.cppvsdbg.console.newExternalWindow.description%"
-                ],
-                "description": "%c_cpp.debuggers.cppvsdbg.console.description%",
-                "default": "internalConsole"
-              },
-              "sourceFileMap": {
-                "type": "object",
-                "description": "%c_cpp.debuggers.sourceFileMap.description%",
-                "default": {
-                  "<sour
```

Comment:
If we're going to change the indentation size for this file, please also update `tools/GenerateOptionsSchema.ts` so that it writes JSON back with 4 spaces instead of 2.

Reply:
Aha.. that's why its getting reset. 

willdo